### PR TITLE
Feat/scout 218 player page tables

### PR DESCRIPTION
**Pull request by dejwi**
### Task Description
[SCOUT-218](https://playmakerpro.atlassian.net/browse/SCOUT-218?atlOrigin=eyJpIjoiNTA3M2YzODQ2NjMyNDBiNmIyNmMxYjlkNDRjNTE1ZGMiLCJwIjoiaiJ9)

on single player page
- added tables with notes, reports, insider notes
- put everything to tabs

What do you think about tabs like this??

### CODE REVIEW

File: src/pages/players/[slug].tsx [hunk 1]
```diff
@@ -36,69 +56,212 @@ export const getServerSideProps = withSessionSsrRole<TData>(
...
+                        )
+                      }
+                      sx={theme => ({
+                        [theme.breakpoints.down('sm')]: {
+                          height: 15,
+                          width: 15,
+                        },
+                      })}
+                    />
+                  )}
```
Comment (jakubfiglak) [inline src/pages/players/[slug].tsx:205]: This doesn't look good here. Let's move this button somewhere else, how about rendering this between tabs bar and the table itself? Or maybe you have some other ideas, I just think the current position doesn't look good and makes this button inaccessible

**Review by jakubfiglak — COMMENTED** [review]
Good work, just one comment regarding positioning of the "add team affiliation" button

**Comment by dejwi** [comment]
What do you think about this connected button?

**Comment by dejwi** [comment]
if you'll like it maybe it's a good idea to style the button on a single team page the same way🤔

**Review by jakubfiglak — APPROVED** [review]

**Pull request merged**